src/MassTransit/Initializers/TypeConverters/EnumTypeConverter.cs

```diff
@@ -119,11 +119,16 @@ public bool TryConvert(ulong input, out T result)
 
         public bool TryConvert(object input, out T result)
         {
+            if (input is string)
```

Comment:
Why not capture the result and use it?

```
if (input is string s)
    return TryConvert(s, out result);
```

Reply:
Because sometime I'm a moron, and still forget that pattern matching is a thing now!

Reply:
No worries, I'm sure I'll see it at some point and auto-fix it.